canonical-web-and-design/snapcraft.io
# Featured snap background colours

Fixes https://github.com/canonicalltd/snap-squad/issues/969

## Done

- Updated `_media-object-snap-partial` include to be the new snap layout
- Updated IOT page with new include
- Updated search results with new include
- Updated publisher pages with new include
- Use client to work out background for featured snaps
- Fallback to #eee if the colour-figurer-outer fails

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/store
- Visit http://0.0.0.0:8004/iot
- Visit http://0.0.0.0:8004/publisher/kde
- Visit http://0.0.0.0:8004/publisher/jetbrains
- Ensure the snap components(?) are consistent throughout
- Ensure the store page has featured banners that have the #eee background
- ...
- **DON'T MERGE THIS PR** waiting on https://bugs.launchpad.net/software-center-agent/+bug/1826006
